api/src/services/job_handler_interface.py

```diff
@@ -13,6 +13,7 @@ class JobStatus(Enum):
     COMPLETED = "completed"
     REMOVED = "removed"
     UNKNOWN = "unknown"
+    SETUP_FAILED = "setup failed"
```

Comment:
Don't like adding new enum values here. We should keep this list as short as possible. How, when, and why it failed can be understood from reading the logs.